### DISCUSSION TITLE
Remove unused images from the File Library

### DISCUSSION THREAD

**Discussion by gabriel-imascono** [directus/directus#2877] [Feature Requests]
Hello, I've searched on Google and here on GitHub but I haven't found this functionality so I guess it doesn't exists, but is there any way to automatically remove all unused images from the File Library?

Thank you.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
